gradle.properties

```diff
@@ -13,11 +13,11 @@ mod_license=GPL 3.0
 author_name=Tourmi
 author_domain=tourmi.dev
 
-minecraft_version=1.21
+minecraft_version=1.21.3
```

Comment:
This file needs to be reverted, otherwise, I'd have to keep track of yet another branch to keep supporting 1.21.

If it's possible to remain compatible with previous minecraft/forge versions, then these values shouldn't ever be changed